prac_06/guitar_test.py

```diff
@@ -0,0 +1,20 @@
+"""
+Test guitars class
+"""
+
+from prac_06.guitars import Guitar
+
+
+def main():
+    name1 = "Gibson L-5 CES"
+    year1 = 1922
+    cost1 = 16035.40
+    guitar = Guitar(name1, year1, cost1)
```

Comment:
Nothing really wrong with this, but you showed an easier way to do this on the line below that avoids multiple variables assignments.

Reply:
Thanks Matt, good idea...